nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -800,29 +816,28 @@ def test_lazy_tractogram_creation(self):
         # Creating LazyTractogram with generators is not allowed as
         # generators get exhausted and are not reusable unlike generator
         # function.
-        assert_raises(TypeError, LazyTractogram, streamlines)
-        assert_raises(TypeError, LazyTractogram,
-                      data_per_point={"none": None})
-        assert_raises(TypeError, LazyTractogram,
-                      data_per_streamline=data_per_streamline)
-        assert_raises(TypeError, LazyTractogram, DATA['streamlines'],
-                      data_per_point=data_per_point)
+        with pytest.raises(TypeError):
+            LazyTractogram(streamlines=streamlines)
+            LazyTractogram(data_per_point={"none": None})
+            LazyTractogram(data_per_streamline=data_per_streamline)
+            LazyTractogram(streamlines=DATA['streamlines'],
+                           data_per_point=data_per_point)
 
         # Empty `LazyTractogram`
         tractogram = LazyTractogram()
         check_tractogram(tractogram)
-        assert_true(tractogram.affine_to_rasmm is None)
+        assert tractogram.affine_to_rasmm is None
 
         # Create tractogram with streamlines and other data
         tractogram = LazyTractogram(DATA['streamlines_func'],
                                     DATA['data_per_streamline_func'],
                                     DATA['data_per_point_func'])
 
-        assert_true(is_lazy_dict(tractogram.data_per_streamline))
-        assert_true(is_lazy_dict(tractogram.data_per_point))
+        assert is_lazy_dict(tractogram.data_per_streamline) is True
+        assert is_lazy_dict(tractogram.data_per_point) is True
```

Comment:
`assert x is True` makes sense if we're testing that `is_lazy_dict` always returns `True`, rather than truthy values. Given that it's not the point of the test, let's drop the `is True`.